test/e2e/cli/pkg_update_test.go

```diff
@@ -54,11 +54,7 @@ var _ = Describe("kraft pkg", func() {
 					Expect(err).ToNot(HaveOccurred())
 
 					Expect(stderr.String()).To(BeEmpty())
-					// The command sends ANSI escape sequences while updating, such as `\e[2K` (erase entire line).
-					// References:
-					//   https://www.regular-expressions.info/nonprint.html
-					//   https://gist.github.com/fnky/458719343aabd01cfb17a3a4f7296797
-					Expect(stdout.String()).To(MatchRegexp(`\x1b\[2K\[\+\] Updating\.\.\. \[\d+\.\d+s\]\r\n`), "Quoted output: %q", stdout)
+					Expect(stdout.String()).To(MatchRegexp(`^{"level":"info","msg":"Updating..."}\n$`))
```

Comment:
nit: We could as well use `Equal()` here if we are gating the string between `^$`.

Reply:
Nit rejected, it makes the test fail with a strange error (the strings are equal), probably whitespace related, but let's just leave it like this

Same for the others